cfgov/cal/views.py

```diff
@@ -122,8 +181,16 @@ def display(request, pdf=False):
             'range_end':range_end,
         })
 
-        if pdf and PDFreactor:
-            return pdf_response(request, context)
+        if pdf:
+            template_name = 'about-us/the-bureau/leadership-calendar/print/index.html'
+            if PDFreactor:
+                return pdf_response(request, context)
```

Comment:
If you don't have PDFreactor installed (for example for local development or if the import fails for some reason), you just get redirected to the main page. Could the `pdf_response` method just return the generic PDF error message in that case instead?